ShareX/Forms/MainForm.cs

```diff
@@ -46,7 +46,6 @@ public partial class MainForm : HotkeyForm
         private bool forceClose, firstUpdateCheck = true;
         private UploadInfoManager uim;
         private ToolStripDropDownItem tsmiImageFileUploaders, tsmiTrayImageFileUploaders, tsmiTextFileUploaders, tsmiTrayTextFileUploaders;
-        private System.Threading.Timer updateTimer;
```

Comment:
OK, will restore later.

From: Jaex [mailto:notifications@github.com]
Sent: 20 April 2015 09:04
To: ShareX/ShareX
Cc: Alan Edwardes
Subject: Re: [ShareX] Amazon Auth V4 Support (#636)


In ShareX/Forms/MainForm.cs<https://github.com/ShareX/ShareX/pull/636#discussion_r28669689>:

> @@ -46,7 +46,6 @@ public partial class MainForm : HotkeyForm

>          private bool forceClose, firstUpdateCheck = true;

>          private UploadInfoManager uim;

>          private ToolStripDropDownItem tsmiImageFileUploaders, tsmiTrayImageFileUploaders, tsmiTextFileUploaders, tsmiTrayTextFileUploaders;

> -        private System.Threading.Timer updateTimer;

It wasn't in use in Debug but in use in Release.

—
Reply to this email directly or view it on GitHub<https://github.com/ShareX/ShareX/pull/636/files#r28669689>.